modules/apps/foundation/users-admin/users-admin-web/src/main/java/com/liferay/users/admin/web/servlet/taglib/ui/OrganizationCategorizationFormNavigatorEntry.java

```diff
@@ -43,15 +43,6 @@ public String getKey() {
 	}
 
 	@Override
```

Comment:
We need to specify the categories when you are creating the organizations if you have a required vocabulary for a Organization, so, we need to display also the categorization when we are creating it.